packages/api/src/taskInfo.ts

```diff
@@ -35,4 +35,6 @@ export interface TaskInfo {
   error?: string;
   progress?: number;
   action?: string;
+  cancellable: boolean;
+  cancellationTokenSourceId?: number;
```

Comment:
we could just have the `cancellationTokenSourceId` ? if it is define, the task is cancellable ?

Reply:
I started and thought about using only this pattern but it was not so clear when consuming